src/background/vvppManager.ts

```diff
@@ -138,22 +145,99 @@ export class VvppManager {
         }
         return parseInt(aMatch[1]) - parseInt(bMatch[1]);
       });
-      for (const p of filePaths) {
-        streams.push(fs.createReadStream(p));
+      format = await this.detectFileFormat(filePaths[0]);
+      if (!format) {
+        throw new Error(`Unknown file format: ${filePaths[0]}`);
       }
+      files = filePaths;
     } else {
       log.log("Not a split file");
-      streams.push(fs.createReadStream(vvppLikeFilePath));
+      format = await this.detectFileFormat(vvppLikeFilePath);
+      if (!format) {
+        throw new Error(`Unknown file format: ${vvppLikeFilePath}`);
+      }
+      files = [vvppLikeFilePath];
     }
 
+    log.log("Format:", format);
```

Comment:
めっちゃ細かいですがこう書くと数行減らせるかもです。
```suggestion
    const format = await this.detectFileFormat(files[0]);
    log.log("Format:", format);
```